### PR TITLE
WT-12480 Remove custom extractors

### PR DESCRIPTION
**Pull request by korteland**

### CODE REVIEW

**Comment by lukech** [comment]
evergreen retry

**Comment by korteland** [comment]
The `clang-analyser` failures seem to be unrelated.

File: dist/api_data.py [hunk 1]
```diff
@@ -454,9 +454,6 @@ def __ge__(self, other):
 ]
 
 index_only_config = [
-    Config('extractor', 'none', r'''
```
Comment (ruby-oujo) [inline dist/api_data.py:457]: Is this config internal only?

**Review by ruby-oujo — APPROVED** [review]

**Comment by ruby-oujo** [comment]
Have one open question, otherwise LGTM.

File: dist/api_data.py (hunk 1 above)
Comment (tod-johnson-mongodb) [reply dist/api_data.py:457]: No, it is not internal only. See https://source.wiredtiger.com/develop/custom_extractors.html#custom_extractors_intro Version 11.3.0 WiredTiger > Reference Guide > Writing WiredTiger applications > Custom Extractors > Introduction to Custom Extractors.

**Review by tod-johnson-mongodb — APPROVED** [review]
LGTM.
I did see Ruby's open issue. What is the response to an unknown configuration item?
There are several evergreen failures.

**Comment by korteland** [comment]
Sorry for hitting merge before submitting this comment - a couple of wires got crossed. For this config, I'd strongly prefer to remove the config than leave it in and potentially leave people erroneously thinking their custom collator is still working.

**Pull request merged**